releng/jenkins/sign-and-deploy/Jenkinsfile

```diff
@@ -96,7 +96,7 @@ spec:
       ''')
     string      (name: 'BRANCH_TO_DEPLOY', defaultValue: 'master', description: 'From which Git branch should the release be created (master for snapshot deployments)?')
     booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_SIGN_JARS', defaultValue: true, description: 'Whether to sign jars using the Eclipse web service')
-    booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_PACK_JARS', defaultValue: true, description: 'Whether to pack jars using pack200')
+    booleanParam(name: 'ORG_GRADLE_PROJECT_OSSPUB_PACK_JARS', defaultValue: false, description: 'Whether to pack jars using pack200')
```

Comment:
maybe we should also completely hardcode this.
i dont know if there is any cbi verifier whatever that still wants packed files